runtime/src/bank/pyth_accumulator.rs

```diff
@@ -0,0 +1,415 @@
+use {
+    super::Bank,
+    crate::accounts_index::{ScanConfig, ScanError},
+    byteorder::LittleEndian,
+    itertools::Itertools,
+    log::*,
+    pyth_oracle::validator::AggregationError,
+    solana_sdk::{
+        account::{AccountSharedData, ReadableAccount},
+        feature_set,
+        hash::hashv,
+        pubkey::Pubkey,
+    },
+    std::{
+        env::{self, VarError},
+        str::FromStr,
+    },
+};
+
+pub const ACCUMULATOR_RING_SIZE: u32 = 10_000;
+
+lazy_static! {
+    static ref ORACLE_PUBKEY: Option<Pubkey> = match env::var("PYTH_ORACLE_PUBKEY") {
+        Ok(value) => Some(
+            value
+                .parse()
+                .expect("invalid value of PYTH_ORACLE_PUBKEY env var")
+        ),
+        Err(VarError::NotPresent) => None,
+        Err(VarError::NotUnicode(err)) => {
+            panic!("invalid value of PYTH_ORACLE_PUBKEY env var: {err:?}");
+        }
+    };
+}
```

Comment:
I suggest making it consistent with other env vars (or changing them to be like this). Also the name `ORACLE_PID` matches the other env vars more.

Reply:
Renamed and unified with other vars.